packages/codec/lib/format/errors.ts

```diff
@@ -258,13 +279,28 @@ export interface FunctionExternalErrorResult {
 
 export type FunctionExternalError = FunctionExternalNonStackPaddingError | FunctionExternalStackPaddingError;
 
+/**
+ * padding error for external function pointer located anywhere other than the stack
+ */
```

Comment:
This sort of doc comment gets listed at the top of the page and so probably should be a complete sentence.
<img width="840" alt="Screen Shot 2019-10-17 at 8 19 17 PM" src="https://user-images.githubusercontent.com/151065/67056759-6e647880-f11b-11e9-8950-dfa3be8c794a.png">